pkg/client/share_manager_client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	rpc "github.com/longhorn/longhorn-share-manager/pkg/rpc"
+	"github.com/longhorn/longhorn-share-manager/pkg/types"
+)
+
+type ShareManagerClient struct {
+	Address string
+}
+
+func NewShareManagerClient(address string) *ShareManagerClient {
+	return &ShareManagerClient{
+		Address: address,
+	}
+}
+
+func (cli *ShareManagerClient) FilesystemTrim(isEncryptedDevice bool) error {
+	conn, err := grpc.Dial(cli.Address, grpc.WithInsecure())
+	if err != nil {
+		return errors.Wrapf(err, "cannot connect share manager service to %v", cli.Address)
```

Comment:
`failed to connect...`

nit: BTW, can we move client to be a field of `ShareManagerClient` instead of recreating it everytime when calling this method?

Reply:
Addressed. Thank you.